bedrock/releasenotes/tests/test_base.py

```diff
@@ -61,6 +62,15 @@ def test_get_release_or_404_esr(self, Q, get_object_or_404):
         Q.assert_any_call(product='Firefox Extended Support Release')
         Q.__or__.assert_called()
 
+    @patch('bedrock.releasenotes.views.get_object_or_404')
+    @patch('bedrock.releasenotes.views.Q')
+    def test_get_release_or_404_endswith_esr(self, Q, get_object_or_404):
+        eq_(views.get_release_or_404('45.0esr', 'Firefox'),
+            get_object_or_404.return_value)
+        Q.assert_any_call(product='Firefox')
+        Q.assert_any_call(product='Firefox Extended Support Release')
+        Q.__or__.assert_called()
```

Comment:
`assert_called` isn't a thing either. I think this whole file may need some work :(